src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/utils/urlHelper.ts

```diff
@@ -8,8 +8,8 @@ export const appPath = `${origin}/${org}/${app}`;
 export const verifySubscriptionUrl = `${origin}/api/v1/${org}/${app}/verifySubscription?partyId=${reportee}`;
 export const languageUrl = `${appPath}/api/Language/GetLanguageAsJSON`;
 export const profileApiUrl = `${appPath}/api/v1/profile/user`;
+export const oldTextResourcesUrl = `${origin}/${org}/${app}/api/textresources`;
```

Comment:
Is this used anywhere?

Reply:
If the New text endpoint isn't available a request will be sent to this endpoint instead. We should be able to remove this once were sure that all apps are running 1.1.0 or later